github-flow.rst

```diff
@@ -0,0 +1,153 @@
+GitHub
+======
+
+Proces komunikacji na GitHubie powinien zakładać jak największe zaufanie
+i odpowiedzialność jej uczestników. Wpłynie to pozytywnie na ich
+zaangażowanie, a w konsekwencji na cały proces komunikacji.
+
+Publikujemy pod swoim nazwiskiem
```

Comment:
Proponuje modyfikacje tytułu rozdziału na "Publikacja przez autora". Powszechne w serwisie GitHub jest wykorzystywanie pseudonimów. Ja także używam go i nie jest to wymagane polityką serwisu. Niezasadne jest oczekiwanie od wszystkich współtwórców, aby tak daleko ustępowali w prywatności. Wydaje się to niecelowe.

Zapraszam do zapoznania się z literaturą, która odnosi się do ingerencji w prywatność poprzez odrzucanie pseudonimów w sieci Internet. Szczególnie polecam https://act.eff.org/action/dear-facebook-authentic-names-are-authentically-dangerous-for-your-users i https://www.eff.org/deeplinks/2015/02/facebooks-name-policy-strikes-again-time-native-americans . Można także dostrzec, że w sekcji "[Anonymity](https://www.eff.org/issues/anonymity)" EFF gromadzi więcej historii, które opisują takie sytuacje.